src/app/tasks/checker-inbox-and-tasks-tabs/loan-disbursal/loan-disbursal.component.ts

```
@@ -156,7 +157,7 @@ export class LoanDisbursalComponent {
       response.forEach((responseEle: any) => {
         if ((responseEle.statusCode = '200')) {
```

Biome (2.1.2)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)